src/components/RequirementView.vue

```diff
@@ -27,8 +27,11 @@
             :subReqIndex="id"
             :subReq="subReq"
             :reqIndex="reqIndex"
+            :toggleableRequirementChoice="toggleableRequirementChoices[subReq.id]"
+            :changeToggleableRequirementChoice="changeToggleableRequirementChoice"
             :color="reqGroupColorMap[req.group][0]"
             :isCompleted="false"
+            @changeToggleableRequirementChoice="changeToggleableRequirementChoice"
```

Comment:
Would we also have this for the completed requirements or is it only for the ongoing ones? 
I tried toggling the option when Option 1 is fulfilled to Option 2, but it doesn't update. But this is expected because it's not saved to Firebase yet right?
![toggleableReqFilled](https://user-images.githubusercontent.com/14082828/98965703-5b67d780-24d8-11eb-9ee0-1b5d3bf2d960.gif)

Reply:
It should be fixed now. Forget there are two subrequirement component in this component.